src/index.ts

```diff
@@ -386,14 +374,20 @@ task("starknet-deploy", "Deploys Starknet contracts which have been compiled.")
 
 async function findPath(traversable: string, name: string) {
     let foundPath: string;
-    await traverseFiles(
-        traversable,
-        file => path.basename(file) === name,
-        async file => {
-            foundPath = file;
-            return 0;
-        }
-    );
+    let paths : string[] = [];
+    if(fs.lstatSync(traversable).isDirectory()){
+        paths = await glob(path.join(traversable,"**","*"));
+    }
+    else{
+        paths.push(traversable);
+    }
+    paths.filter(file => {
+        return path.basename(file) === name;
+    }).forEach(file => {
+        foundPath = file;
+        return 0;
```

Comment:
No need to return the `0`. That used to be so just to comply with what `traverseFiles` expected - it expected a number as a return value, so it was given a symbolic zero.
Now that I think about it, this is how it should be implemented: it should be checked whether `paths` contains just one path, throw if it is not so (with a message saying it's ambiguous), and return the zeroth element.